pb/lmsysmd/load/v1/model.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package lmsysmd.load.v1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+service ModelService {
+  rpc BatchCreateModel(BatchCreateModelRequest) returns (google.protobuf.Empty) {
+    option idempotency_level = IDEMPOTENT; 
+  }
+}
+
+message BatchCreateModelRequest {
+  string google_sheet_url = 1 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true,
+    (buf.validate.field).string.min_len = 2,
+    (buf.validate.field).string.max_len = 60
+  ];
+}
+
+message Model {
+  ModelId model_id = 1 [ 
+    (buf.validate.field).required = true, 
+    (buf.validate.field).enum.defined_only = true 
+  ];
+  string display_name = 2 [ 
+    (buf.validate.field).required = true, 
+    (buf.validate.field).string.min_len = 2,
+    (buf.validate.field).string.max_len = 20
```

Comment:
Might need a larger buffer here. Some models can exceed this length, especially if we are including the release date.